vector/v.category/v.category.md

```diff
@@ -128,6 +151,85 @@ Print vector categories from the first layer, only for feature ids 1-50.
 v.category input=roads option=print layer=1 id=1-50
 ```
 
+Print vector categories from the first layer, only for feature ids 1-50 in JSON format.
+
+```sh
+v.category input=roads option=print layer=1 id=1-50 format=json
+```
+
+### Print only layer numbers in JSON format
+
+```sh
+v.category input=roads option=layers format=json
+```
+
+### Using v.category JSON output with pandas
+
+Using report option in JSON format with pandas:
+
+```python
+  import json, io
+  import grass.script as gs
+  import pandas as pd
+
+  # Run v.category command with report option.
+  data = gs.read_command(
```

Comment:
When you use parse_command, it understands format=json and will return dictionary:

```
report = gs.parse_command("v.category", input="test", format="json", option="report")
pd.DataFrame(report)
```